conda-forge.yml

```diff
@@ -1 +1,5 @@
-[]
\ No newline at end of file
+appveyor:
+  secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
+travis:
+  secure: {BINSTAR_TOKEN: KPrVIfxOmlAjPFU13v0Z8+Ijyqep1NfZOhPY90PYkYz1YOHMBDpn/fHbsgfQKTfm3tzvR+3rzjpPmR3hVhMFfR7RoIjImikHj+XgKQ1bgtmreZeKd8AeKvN4uI0ZcSR+Ehg21MdZ/8+pEDtHsZ5fROqUI2peHWqKyvzzSYwbrb/aKN5B8EjIRPYI7CV58CdQY/oEbtgE87t8LSIPMvk25xlStI6zRIanfKvyFJJawdsWY8e0UEzYWOJVrWxgH1Bg+f9xj/oEvUKl5/fXlhHoIOBWSP5I/qMxEnq+9s/XZ3wwiFbXQ5lJPwui53RrtpWv/9mpHvOP3+GE3rigWv4leNSTIkakbFnW5h5oLV6/ZgxAzQmm4/zXcPjNPeK1DMcBX5Cii9mvptdon2rhxbxppHb81TfiRv0qB5QRa/oLFGq1TWLoP1zW9rlem06jeYPGAd2XXzEYeXfgB7uwh7BgS+bv20LtNArnxA0vsGJiBEq222NimDjCQpU/qwH1UIITGog6v+HPvMtMghYq3Z2hF5slpFgVHkO23EVrA6FAVvsqVsdZbw9mRy3ReZFpuKwNKavI/2MEq3dCpAfR39Gu/wLvUy+IwIN2PgXrHE9hNJeT0CBu9m/3Hig819hPGhclXJuJTHWnbubbyYIHZLxd2VINEXe7UFqowGlBP+DOv78=}
+provider: {win: azure}
```

Comment:
@ocefpaf, does this seem okay to do?  Again, I'm not sure why the bot didn't take care of it but this bit seems to be in all the `conda-forge.yml` files I could find when I did a little hunting around.

Reply:
Sure. I'll investigate what is going on later but this should fix it. Thanks!